polaris-core/src/main/java/org/apache/polaris/core/persistence/dao/CredentialVendorDao.java

```diff
@@ -16,17 +16,18 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.polaris.core.storage;
+package org.apache.polaris.core.persistence.dao;
 
 import jakarta.annotation.Nonnull;
 import java.util.Set;
 import org.apache.polaris.core.PolarisCallContext;
 import org.apache.polaris.core.entity.PolarisEntityType;
 import org.apache.polaris.core.persistence.dao.entity.ScopedCredentialsResult;
 import org.apache.polaris.core.persistence.dao.entity.ValidateAccessResult;
+import org.apache.polaris.core.storage.PolarisStorageActions;
 
 /** Manage credentials for storage locations. */
-public interface PolarisCredentialVendor {
+public interface CredentialVendorDao {
```

Comment:
I'm not sure how to interpret this rename (and relocation to a different package).

@flyrain : could you share an outline of the general ideas behind this refactoring on the dev ML? thx!